cli/tests/environment-activate.bats

```diff
@@ -375,3 +375,30 @@ env_is_activated() {
   assert_success
   assert_output --regexp "source .*/activate/zsh"
 }
+
+# bats test_tags=activate,activate:inplace-modifies
+@test "'flox activate' allows to modify current shell" {
```

Comment:
nit: other tests in the file are split up a bit more granularly. I don't particularly care but I thought you and @zmitchell sometimes prefer that 🤷

Reply:
split them up...